lib/demon/sidekiq.rb

```diff
@@ -3,6 +3,49 @@
 require "demon/base"
 
 class Demon::Sidekiq < Demon::Base
+  RANDOM_HEX = SecureRandom.hex
+
+  def self.heartbeat_queues_list_key
+    @@heartbeat_queues_list_key ||= "#{RANDOM_HEX}_heartbeat_queues_list"
+  end
+
+  def self.queues_last_heartbeat_hash_key
+    @@queues_last_heartbeat_hash_key ||= "#{RANDOM_HEX}_queues_last_heartbeat_hash"
+  end
+
+  def self.set_queue_heartbeat(name)
```

Comment:
I think we should rename this method, perhaps `trigger_heartbeat`. When I think `set_xyz` it sounds to me like replacing the member `xyz` with something else.